### PR TITLE
Uni build

### PR DESCRIPTION
**Pull request by mbouzi**
- universal login page at "/satellites"

### CODE REVIEW

File: assets/scss/_universal-login.scss [hunk 1]
```diff
@@ -35,6 +28,8 @@ body {
     letter-spacing: -0.100741px;
     color: #252525;
     user-select: none;
+    font-family: 'Inter', sans-serif;
+
```
Comment (VitaliiShpital) [inline assets/scss/_universal-login.scss]: redundant line

File: layouts/universal-login/single.html [hunk 2]
```diff
@@ -64,6 +63,70 @@ <h1 class="title">Select Your Satellite</h1>
...
     </div>
-    <script type="text/javascript" src="/static/login.js"></script>
+    {{partial "site-scripts" .}}
 </body>
-</html>
\ No newline at end of file
+</html>
+
+<script type="text/javascript">
+  class Satellite {
```
Comment (VitaliiShpital) [inline layouts/universal-login/single.html]: use 4 spaces indents inside script tag

File: layouts/universal-login/single.html [hunk 3]
```diff
@@ -64,6 +63,70 @@ <h1 class="title">Select Your Satellite</h1>
...
+      asiaSatellite.unselectSatellite();
+
+      europeSatellite.selectSatellite();
+      setIframeSource('https://europe-west-1.tardigrade.io/login');
+  }
+
+  function setIframeSource(url) {
+      document.getElementById('iframe').src = url;
+  }
+
```
Comment (VitaliiShpital) [inline layouts/universal-login/single.html:127]: redundant line

File: assets/scss/_universal-login.scss (hunk 1 above)
Comment (mbouzi) [reply assets/scss/_universal-login.scss]: @VitaliiShpital haven't been able to figure out why this font doesn't get applied here if this line is not here

File: layouts/universal-login/single.html (hunk 3 above)
Comment (mbouzi) [reply layouts/universal-login/single.html:127]: @VitaliiShpital not seeing where this line exists previously, it looks like this function is declared once and being used a few times. Did you want me to remove this function and all instances of it?

File: layouts/universal-login/single.html (hunk 2 above)
Comment (mbouzi) [reply layouts/universal-login/single.html]: done!

File: layouts/universal-login/single.html (hunk 3 above)
Comment (VitaliiShpital) [reply layouts/universal-login/single.html:127]: @mbouzi no, i just meant that there is redundant empty line before script closing tag. That's all

Same as my first comment

**Pull request merged**